src/iconid.rs

```diff
@@ -40,14 +51,31 @@ impl IconIdentifier {
                 .map_err(IconResolutionError::ReadError)?
                 .map_codepoint(*cp)
                 .ok_or(IconResolutionError::NoCmapEntry(*cp)),
-            IconIdentifier::Name(name) => resolve_ligature(font, name.as_str()),
+            IconIdentifier::Name(name) => font.resolve_ligature(name.as_str()),
         }?;
 
         apply_location_based_substitution(font, location, gid)
             .map_err(IconResolutionError::ReadError)
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct Icon {
+    name: String,
+    codepoints: Vec<u32>,
```

Comment:
Won't this just be the codepoints in name?

Reply:
per f2f it's specifically the PUA codepoint(s). Although ... I think for one name there should be exactly 1 PUA, multiple PUA would mean multiple names as well?